prog/vm/nexus.rb

```diff
@@ -130,7 +130,7 @@ def self.assemble_with_sshable(unix_user, *, **kwargs)
     ssh_key = SshKey.generate
     kwargs[:unix_user] = unix_user
     st = assemble(ssh_key.public_key, *, **kwargs)
-    Sshable.create(unix_user: unix_user, host: "temp_#{st.id}", raw_private_key_1: ssh_key.keypair) {
+    Sshable.create(unix_user: (unix_user == "runneradmin") ? "runneradmin" : "rhizome", host: "temp_#{st.id}", raw_private_key_1: ssh_key.keypair) {
```

Comment:
That is a little hacky: `(unix_user == "runneradmin")`

I'm not sure if it's feasible to use the rhizome user for Github runners. We could maybe extend `assemble_with_sshable` to accept the sshable user as additional parameter.

Reply:
naaah let's leave runneradmin, that IIRC is named to be the same as the image and we don't need a drip of bug reports about that.

I think more irksome is that the unix_user is ignored for every other value. I guess the problem is Util.rootish_ssh has no viable way to use a non-default for this? I will analyze this more closely. Thank you for testing & writing the patch versions

Reply:
we could maybe do something like: `assemble_with_sshable(unix_user, sshable_user, ...`
assemble_with_sshable("ubi", "rhizome", ...)
assemble_with_sshable("runneradmin", "runneradmin", ...)

Reply:
See https://github.com/ubicloud/ubicloud/pull/2979#issuecomment-2731376582, maybe there's a loose connection somewhere, but it seems like there's already two places to feed in users, one in the `bud` of Bootstrap and the other via the `assemble_with_sshable`